WordPress/Classes/Extensions/UIAlertController+Helpers.swift

```diff
@@ -16,7 +16,42 @@ extension UIAlertController {
         while leafViewController.presentedViewController != nil && !leafViewController.presentedViewController!.isBeingDismissed {
             leafViewController = leafViewController.presentedViewController!
         }
-
         leafViewController.present(self, animated: true, completion: nil)
     }
 }
+
+//MARK: - copy text to Clipboard
+extension UIAlertController {
+    /// This method is used for presenting the Action sheet
+    /// for copying text to clipboard. The action sheet has 2 options:
+    /// copy: will copy the text to the clipboard
+    /// cancel: dismiss the action sheet
+    @objc static func copyTextAlertController(_ text: String?,
+                                              completion: ((Bool) -> Void)? = nil) -> UIAlertController? {
+        guard let text = text else {
```

Comment:
This is sorta-personal-nitpick, sorta-actual-guideline we have (
https://github.com/wordpress-mobile/swift-style-guide#deal-with-optionals-early), but it seems pointless for this method to handle optional `text` arguments if the only thing it does is immediately return with `false` in the callback. It would be better for the callers the handle the `nil` `text` parameters and react appropriately.